bindings/pyroot_experimental/PyROOT/python/JsMVA/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
```

Comment:
Same comment as for the CMakeLists.txt of JupyROOT, better in a section of the PyROOT CMakeLists.